### PR TITLE
[Filter/Caffe2] add caffe2 at the nnstreamer build process- @open sesame 6/18 15:55

### PR DESCRIPTION
**Pull request by helloahn**
make nnstreamer build process including the caffe2.

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by helloahn** [comment]
@myungjoo 
it's included at `pytorch` and it is already added at `debian/control`.
the reason why broken of CI is, our ppa is not supplying the newest `pytorch` package.
@kparichay is solving it.

**Comment by myungjoo** [comment]
> @myungjoo
> it's included at `pytorch` and it is already added at `debian/control`.
> the reason why broken of CI is, our ppa is not supplying the newest `pytorch` package.
> @kparichay is solving it.

So.. ```pytorch``` has all in it, including devel files?

Anyway, it appears that it is because @kparichay didn't publish the resulting binaries to nnstreamer ppa, but to his own ppa. I've resubmitted a build request for nnstreamer ppa.

**Comment by myungjoo** [comment]
You probably need:
```
$ sudo apt update
```
in the CI server?

File: ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc [hunk 1]
```diff
@@ -437,3 +448,13 @@ caffe2_core_run (void * caffe2, const GstTensorMemory * input,
   Caffe2Core *c = (Caffe2Core *) caffe2;
   return c->run (input, output);
 }
+
+/**
+ * @brief	the destroy notify method for tensorflow. it will free the output tensor
```
Comment (kparichay) [inline ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: tensorflow -> caffe2?

Comment (helloahn) [reply ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: it's fixed :)

**Pull request merged**